config/config.exs

```diff
@@ -301,6 +301,9 @@ config :ex_cldr,
 config :oli, :datashop,
   cache_limit: String.to_integer(System.get_env("DATASHOP_CACHE_LIMIT", "200"))
 
+config :oli, :student_sign_in,
```

Comment:
This fix also has to add a `runtime.exs` equivalent.  That will allow us to drive this from the environment variable at runtime.  config.exs only drives a compile time change.